module1/src/com/epam/linearprograms/HomeTask_1_1.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.linearprograms;
+
+public class HomeTask_1_1 {
+    public static void main(String [] args){
+        int a = 5;
+        int b = 27;
+        int c = 33;
+        int z = ((a - 3) + 6) / 2 + c;
+         System.out.print(z);
```

Comment:
write it into 1 line:
```suggestion
        System.out.print("Value of z = " + ((a - 3) + 6) / 2 + c);
```